### PR TITLE
Add gateway/firewall name to vlan detail and list command

### LINKED ISSUE #1480: Correctly indicate when a VLAN is inside a firewall
> Reminder: No username or APIkeys should be added to these issues, as they are public.

**Describe the bug**
`slcli` shows `firewall : No` on VLANs that do have a firewall

```
~ $ slcli -v vlan list -d dal13                                                                                                                                                                 16:37:29
Calling: SoftLayer_Account::getNetworkVlans(id=None, mask='firewallInterfaces,hardwareCount,primaryRouter[id, fullyQualifiedDomainName, datacenter],subnetCount,totalPrimaryIpAddressCount,virtualGuestCount,networkSpace', filter='{'networkVlans': {'primaryRouter': {'datacenter': {'name': {'operation': '_= dal13'}}}}}', args=(), limit=100, offset=0))
:.........:........:......................:..........:............:..........:.................:............:
:    id   : number :         name         : firewall : datacenter : hardware : virtual_servers : public_ips :
:.........:........:......................:..........:............:..........:.................:............:
: 1687473 :  845   :          -           :    No    :   dal13    :    0     :        1        :    125     :
: 1901043 :  827   :        dh_ics        :    No    :   dal13    :    3     :        11       :    1021    :
: 1901055 :  767   :          -           :    No    :   dal13    :    3     :        10       :    1021    :
: 2272683 :  1145  :          -           :    No    :   dal13    :    0     :        0        :     0      :
: 2272685 :  906   :          -           :    No    :   dal13    :    0     :        0        :     0      :
: 2461431 :  1171  :          -           :    No    :   dal13    :    0     :        9        :     42     :
: 2461433 :  1309  :          -           :    No    :   dal13    :    0     :        10       :     61     :
: 2611043 :  1579  :   robtest-private    :    No    :   dal13    :    0     :        3        :     61     :
: 2618107 :  2688  :     d13_pub_bnpp     :    No    :   dal13    :    0     :        6        :     13     :
: 2619303 :  3788  :   dal13_priv_bnpp    :    No    :   dal13    :    0     :        0        :     0      :
: 2627161 :  3819  :          -           :    No    :   dal13    :    1     :        0        :     61     :
: 2627163 :  2726  :          -           :    No    :   dal13    :    1     :        0        :     5      :
: 2648937 :  869   :          -           :    No    :   dal13    :    0     :        0        :     0      :
: 2648941 :  1770  :          -           :    No    :   dal13    :    0     :        1        :     61     :
: 2897860 :  3308  : pthorvlanprvdallas13 :    No    :   dal13    :    0     :        0        :     0      :
: 2897876 :  2841  : pthorvlanpubdallas13 :    No    :   dal13    :    0     :        0        :     0      :
: 3039087 :  905   :          -           :    No    :   dal13    :    0     :        3        :     13     :
: 3039089 :  908   :          -           :    No    :   dal13    :    0     :        3        :     61     :
:.........:........:......................:..........:............:..........:.................:............:
:..................:.........................................................................:
:       name       :                                  value                                  :
:..................:.........................................................................:
:  execution_time  :                                1.730383s                                :
:    api_calls     :              SoftLayer_Account::getNetworkVlans (1.725769s)             :
:     version      :                         softlayer-python/v5.9.4                         :
:  python_version  :                 3.9.4 (default, Apr 28 2021, 17:20:42)                  :
:                  :                  [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]                 :
: library_location : /home/scott/.pyenv/versions/3.9.4/lib/python3.9/site-packages/SoftLayer :
:..................:.........................................................................:
```

**Expected behavior**
For example, `1901055` has a firewall, so I would expect it to say `Yes`

**Screenshots**
image

**Version**
```
~ $ slcli --version                                                                                                                                                                             16:38:34
Current: slcli (SoftLayer Command-line) v5.9.4
Latest:  slcli (SoftLayer Command-line) v5.9.4
```

### PR DESCRIPTION
**Pull request by ATGE**
fixes #1480

### CODE REVIEW

**Comment by ATGE** [comment]
### Testing 

- ✔️slcli vlan detail 2282897 | grep Gateway
      Gateway/Firewall  support    
                                                        
- ✔️ slcli vlan detail 361652 | grep Firewall
      Gateway/Firewall  firewall-vlan.networklayer.com  

- ✔️$slcli vlan list

**Review by allmightyspiff — APPROVED** [review]
Looks good, thanks

**Review by caberos — APPROVED** [review]
LGTM.

**Review by FernandoOjeda — COMMENTED** [review]
Looks good, just check my comment on the network manager.

File: SoftLayer/managers/network.py [hunk 1]
```diff
@@ -432,6 +436,16 @@ def get_vlan(self, vlan_id):
         """
         return self.vlan.getObject(id=vlan_id, mask=DEFAULT_GET_VLAN_MASK)
 
+    def get_network_gateway_firewall(self, vlan_id):
```
[the remark this answers is not in the document]
Comment (allmightyspiff) [reply SoftLayer/managers/network.py:439]: I think this method could just be removed entirely, it doesn't look like its actually called anywhere to me. And if it is called somewhere, please replace it with `get_vlan` as Fernando suggested. Thanks

**Review by allmightyspiff — CHANGES_REQUESTED** [review]

**Review by ATGE — COMMENTED** [review]
### TESTED:
✔️slcli vlan detail 2282897 | grep Gateway
Gateway/Firewall support

✔️ slcli vlan detail 361652 | grep Firewall
Gateway/Firewall firewall-vlan.networklayer.com

✔️$slcli vlan list

File: SoftLayer/managers/network.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ATGE) [reply SoftLayer/managers/network.py]: `get_network_gateway_firewall` was removed 👍

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**